libs/platform/src/lib/table/table.component.ts

```diff
@@ -254,6 +256,10 @@ export class TableComponent<T = any> extends Table implements AfterViewInit, OnD
     @Input()
     semanticHighlighting: string;
 
+    /** String or function to calculate additional rows' CSS classes. */
+    @Input()
+    rowsClass: TableRowClass<T>;
```

Comment:
Two comments/questions: in keeping with the rest of the library I think just naming the input `class` would be fine, and I am curious what is the advantage of exporting a new type for something that is simply a string?

Reply:
This is the input for the table so just `class` won't say too much about the purpose and the purpose to have the ability to set classes for the table rows.

It can also be a function so the customer can provide custom logic to define CSS classes that's why the type is created.